README.md

```diff
@@ -911,6 +912,36 @@ Finally, to start a task that is not started by default, use the `-s` flag.
 
 
 
+### `humility log`
+
+`humility log` reads and displays any Hubris string buffers (as created
+via the `stringbuf!` macro in the Hubris `ringbuf` crate).  e.g.:
+
+```console
+% humility log -m LOG_RINGBUF
+humility: attached via J-Link
+humility: ring buffer task_jefe::LOG_RINGBUF in jefe:
+instruction
+Task #2 Memory fault at address 0x0
+Task #2 Illegal instruction
+Task #2 Memory fault at address 0x0
+Task #2 Illegal instruction
+Task #2 Memory fault at address 0x0
+Task #2 Illegal instruction
+...
+```
+
+Use `-m` or `--monitor` to continuously monitor the buffer, otherwise will just print the
+current log and exit.
```

Comment:
Probably should add a line about the delay as well.

Reply:
Good Idea